src/components/TodoList/TodoList.tsx

```diff
@@ -1,224 +1,120 @@
 /* eslint-disable */
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getTodos } from '../../api';
+import { todosSlice } from '../../features/todos';
+import cn from 'classnames';
+import { currentTodoSlice } from '../../features/currentTodo';
 import React from 'react';
 
+function filterTodos(todos: any, query: string, status: string) {
+  let filteredTodos = todos;
+  if (query) {
+    filteredTodos = todos.filter((todo: any) =>
+      todo.title.toLowerCase().includes(query.toLowerCase()),
+    );
+  }
+  if (status !== 'all') {
```

Comment:
```suggestion
  if (status !== Status.All) {
```
in case you've created enum for it you should use it everywhere